frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/artifacts.cy.ts

```diff
@@ -213,6 +213,38 @@ describe('Artifacts', () => {
         );
     });
   });
+  describe('Pipeline run visualization tab when ArtifactApi feature flag is enabled', () => {
+    beforeEach(() => {
+      initPipelineTopologyIntercepts({ disableArtifactsAPI: false, disableS3Endpoint: false });
+      cy.interceptOdh(
+        'GET /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/artifacts/:artifactId',
+        {
+          query: { view: 'DOWNLOAD' },
+          path: { namespace: projectName, serviceName: 'dspa', artifactId: 18 },
+        },
+        mockArtifactStorage({ namespace: projectName, artifactId: '18' }),
+      );
+      cy.intercept(
+        'GET',
+        'https://test.s3.dualstack.us-east-1.amazonaws.com/metrics-visualization-pipeline/5e873c64-39fa-4dd4-83db-eff0cdd1e274/html-visualization/html_artifact?X-Amz-Algorithm=AWS4-HMAC-SHA256\u0026X-Amz-Credential=AKIAYQPE7PSILMBBLXMO%2F20240808%2Fus-east-1%2Fs3%2Faws4_request\u0026X-Amz-Date=20240808T070034Z\u0026X-Amz-Expires=15\u0026X-Amz-SignedHeaders=host\u0026response-content-disposition=attachment%3B%20filename%3D%22%22\u0026X-Amz-Signature=de39ee684dd606e75da3b07c1b9f0820f7442ea7a037ae1bffccea9e33610ea9',
+        '<html>helloWorld',
```

Comment:
no matching end tag?

Reply:
Thanks for catching this, I will update it